system-variables.md

```diff
@@ -1,129 +1,838 @@
 ---
-title: The System Variables
-summary: Learn how to use the system variables in TiDB.
+title: System Variables
+summary: Use system variables to optimize performance or alter running behavior.
 category: reference
-aliases: ['/docs/stable/system-variables/','/docs/v4.0/system-variables/','/docs/stable/reference/configuration/tidb-server/mysql-variables/']
+aliases: ['/docs/stable/system-variables/','/docs/v4.0/system-variables/','/docs/stable/reference/configuration/tidb-server/mysql-variables/','/docs/stable/tidb-specific-system-variables/','/docs/v4.0/tidb-specific-system-variables/','/docs/stable/reference/configuration/tidb-server/tidb-specific-variables/']
```

Comment:
```suggestion
aliases: ['/tidb/stable/tidb-specific-system-variables','/docs/stable/system-variables/','/docs/v4.0/system-variables/','/docs/stable/reference/configuration/tidb-server/mysql-variables/','/docs/stable/tidb-specific-system-variables/','/docs/v4.0/tidb-specific-system-variables/','/docs/stable/reference/configuration/tidb-server/tidb-specific-variables/']
```
I just realized we forgot this alias: '/tidb/stable/tidb-specific-system-variables'. It is to redirect the deleted file to system-variables.md so no dead link will be caused by the deletion. @nullnotnil

Reply:
I'll add this `'/tidb/dev/tidb-specific-system-variables'` alias in [my PR](https://github.com/pingcap/docs/pull/3239).

Reply:
Added it in https://github.com/pingcap/docs/pull/3239/commits/1c5608e2117f48f0c2ba81ed7470948ecc9997cb